build.gradle

```diff
@@ -736,123 +763,58 @@ allModuleNames.split(',').each{ moduleName ->
             baseName moduleName
         }
 
-      	if (isExternal.toBoolean()){
-      		tasks["dist_$moduleName"].dependsOn("jar_$moduleName")
-      	}
+        if (isExternal.toBoolean()) {
+            tasks["dist_$moduleName"].dependsOn("jar_$moduleName")
+        }
     }
 
-    if (gwtModule != null){
-        sourceSets.create(moduleName+'_gwt')
-        sourceSets[moduleName+'_gwt'].java.srcDirs=files("$srcGwtDir")
-        sourceSets[moduleName+'_gwt'].resources.srcDirs=files("$srcGwtDir")
-        task "gwt_$moduleName" (dependsOn: gwtClasses, type: JavaExec) {
-            ext.buildDir =  project.buildDir.toString()  +"/gwt/$moduleName"
-            ext.extraDir =  project.buildDir.toString() + "/extra/$moduleName"
-            inputs.files sourceSets.gwt.java.srcDirs
-            inputs.dir sourceSets.gwt.output.resourcesDir
-            outputs.dir buildDir
-
-            // Workaround for incremental build (GRADLE-1483)
-            outputs.upToDateSpec = new org.gradle.api.specs.AndSpec()
-
-            doFirst {
```

Comment:
`doFirst` **inhibits paralellization** of tasks.

From https://discuss.gradle.org/t/can-we-run-tasks-in-parallel-in-single-build-project/18321:

> Any task that has custom actions (i.e. ones added via `org.gradle.api.Task#doLast(org.gradle.api.Action)` or `org.gradle.api.Task#doFirst(org.gradle.api.Action)` is not considered parallelizable even if its type carries this annotation. This is because it cannot be known whether the added action is parallel safe or not.